src/main/scala/io/citrine/lolo/trees/classification/ClassificationTrainingLeaf.scala

```diff
@@ -0,0 +1,35 @@
+package io.citrine.lolo.trees.classification
+
+import io.citrine.lolo.{Learner, TrainingResult}
+import io.citrine.lolo.trees.TrainingLeaf
+import io.citrine.random.Random
+
+import scala.collection.mutable
+
+/** Training leaf for a classification tree. */
+case class ClassificationTrainingLeaf(
+    trainingData: Seq[(Vector[AnyVal], Char, Double)],
+    trainingResult: TrainingResult,
+    depth: Int
+) extends TrainingLeaf[Char] {
+
+  override def getFeatureImportance(): mutable.ArraySeq[Double] = mutable.ArraySeq.fill(trainingData.head._1.size)(0.0)
```

Comment:
Should we rename this to just `featureImportance` to follow the other style changes?

Reply:
Also, why is this and associated methods on multi-task returning a `mutable.ArraySeq[Double]`. Was it always like that? If there's a strong reason for this collection choose we should keep it, but if not, i think we should use a standard immutable vector as we do elsewhere.

Reply:
It was always like this because the feature importance is built up by starting with an array of 0s and repeatedly adding contributions, then normalizing at the end.

Reply:
> Should we rename this to just featureImportance to follow the other style changes?

yes